gsheets-workflow-api/notebooks/exploration-notebook.py

```diff
@@ -26,10 +26,19 @@
 app_logger.setLevel(logging.DEBUG)
 app_logger.debug("test")
 
+# +
+from lib.config import read_config
+
+config = read_config()
+# -
+
+api_tokens = config['SURVEY_MONKEY_API_TOKEN'].split(';')
+api_token = api_tokens[0]
```

Comment:
Add comment here that only the first api token is used and how to use the second/third one

Reply:
added: https://github.com/Gapminder/gapminder-igno-survey-processing/blob/c9923004394b9e2ed1434dd991986874124abb7b/gsheets-workflow-api/notebooks/exploration-notebook.py#L35-L41